theatre/studio/src/panels/OutlinePanel/BaseItem.tsx

```diff
@@ -110,12 +101,16 @@ const Head_Icon_WithDescendants = styled.span<{isOpen: boolean}>`
   font-size: 9px;
   position: relative;
   display: block;
+
+  transform: rotate(${(props) => (props.isOpen ? '0deg' : '-90deg')});
```

Comment:
perf: pull this out into a style prop so that styled does not recompute the css when this changes.